testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -982,7 +982,7 @@ public void testCreateAndDropTableWithSameLocationFailsOnTrino(int specVersion)
         onTrino().executeQuery(format("DROP TABLE %s", trinoTableName(tableSameLocation1)));
 
         assertQueryFailure(() -> onTrino().executeQuery(format("SELECT * FROM %s", trinoTableName(tableSameLocation2))))
-                .hasMessageMatching(".*Failed to open input stream for file.*");
+                .hasMessageMatching(".*Metadata not found in metadata location for table (.*).*");
```

Comment:
Does the exception still include the file on which we failed, along with the original exception?
let's test for that